src/PagesBody/ProfilePage/ProfilePage.js

```diff
@@ -1,16 +1,19 @@
 import propTypes from 'prop-types';
+import { useContext } from 'react';
 
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
 import FakeButton from '../../Atoms/FakeButton/FakeButton';
 import ProfileCard from '../../Molecules/ProfileCard/ProfileCard';
+import { UserDataContext } from '../../Context/UserDataContext';
 
-const ProfilePage = ({ userId }) => {
+const ProfilePage = () => {
+  const userData = useContext(UserDataContext);
   return (
     <ColumnWrapper paddingLeftRight={1}>
-      <ProfileCard />
+      <ProfileCard userData={userData.state} />
```

Comment:
If you need the context data in the other component, use the context over there, no need to pass down data that is already stored in context.

Reply:
Thanx for this comment. Refactored